discovery.md

```diff
@@ -808,79 +806,73 @@ Content-Type: application/json
 }
 ```
 
-The format of the HTTP Response MUST adhere to the following:
+The format of the `200 OK` HTTP Response MUST adhere to the following:
 ```
+200 OK
 Content-Type: application/json
 
 {
-  "url": "{url}",
+  "id": "{id}",
```

Comment:
If id/epoch are good enough for POST responses, it seems like it should be good enough for DELETE too

Reply:
As [noted](https://github.com/cloudevents/spec/pull/837/files#r670146051), I [asked about](https://github.com/cloudevents/spec/pull/837/files#r664990150) whether these should be consistent.  The thought basis for returning the entire service in response to a DELETE is that one can do that only providing the `id` and optionally `epoch` but may not have the rest of the information in the service.  I'm not feeling super strong about this supporting vital use cases.  However, if we went that route, as commented, it may be appropriate to respond with the old values of a service in cases where it is overwritten as opposed to created.  I would prefer that one of these strategies should be selected and that we should be consistent about it.  Sorry to ask the ether but I know I'm not holding all the history and every context so I aimed for a middle ground.